Mathlib/Analysis/LConvolution.lean

```diff
@@ -52,13 +52,13 @@ noncomputable def mlconvolution (f g : G → ℝ≥0∞) (μ : Measure G) :
 
 /-- Scoped notation for the multiplicative convolution of functions with respect to a measure `μ`.
 -/
-scoped[MeasureTheory] notation:67 f " ⋆ₘₗ["μ:67"] " g:66 => MeasureTheory.mlconvolution f g μ
+scoped[MeasureTheory] notation:67 f " ⋆ₘₗ[" μ:67 "] " g:66 => MeasureTheory.mlconvolution f g μ
```

Comment:
This change does not change the pretty-printing, just the formatting of this line.